dashboard.py

```diff
@@ -0,0 +1,177 @@
+from flask import jsonify, render_template, request, make_response, Blueprint, url_for, session, redirect
+import json
+from utils.make_session import make_session
+import rethinkdb as r
+import hashlib
+from random import randint
+
+config = json.load(open('config.json'))
+
+dash = Blueprint('dashboard', __name__, template_folder='views', static_folder='views/assets')
+
+API_BASE_URL = 'https://discordapp.com/api'
+AUTHORIZATION_BASE_URL = API_BASE_URL + '/oauth2/authorize'
+TOKEN_URL = API_BASE_URL + '/oauth2/token'
+RDB_ADDRESS = config['rdb_address']
+RDB_PORT = config['rdb_port']
+RDB_DB = config['rdb_db']
+
+rdb = r.connect(RDB_ADDRESS, RDB_PORT)
```

Comment:
`db=RDB_DB`, this removes the need to do `r.db(RDB_DB).table(...)` on each call.